pegasus/sites.v3/code.org/public/congrats/splat.haml

```diff
@@ -0,0 +1,31 @@
+---
+title: Congratulations
+theme: responsive
+footer: suppl_footer
+social:
+  "twitter:card": player
+  "twitter:site": "@codeorg"
+  "twitter:url": "https://code.org/"
+  "twitter:title": "The Hour of Code is here"
+  "twitter:description": "The Hour of Code is a global movement reaching tens of millions of students in 180+ countries and over 45 languages. Ages 4 to 104."
+  "twitter:image:src": "http://csedweek.org/images/code-video-thumbnail.jpg"
+  "twitter:player": "https://www.youtube.com/embed/rH7AjDMz_dc?iv_load_policy=3&rel=0&autohide=1&showinfo=0"
+  "twitter:player:width": 1920
+  "twitter:player:height": 1080
+max_age: 60
```

Comment:
It's a caching thing. I don't know exactly what 60 means (60 days?), but it's used on the existing /congrats page so I figured it could follow the same behavior. I could remove it, since I don't think these congrats pages are as important as the Hour of Code congrats page.